gui/kivy/uix/ui_screens/about.kv

```diff
@@ -28,8 +28,7 @@ Popup:
                 markup: True
                 text: '[color=6666ff][ref=x]https://electrum.org[/ref][/color]'
                 on_ref_press:
-                    import webbrowser
-                    webbrowser.open("https://electrum.org")
+                    1/0
```

Comment:
revert testing code

Reply:
Whoops, thanks! Although the original code also crashes the app but sadly not within Python.